blockem/blockem.php

```diff
@@ -137,20 +139,24 @@ function blockem_prepare_body_content_filter(\Friendica\App $a, &$hook_data)
 	}
 }
 
-function blockem_display_item(&$a,&$b) {
-	if(strstr($b['output']['body'],'id="blockem-wrap-'))
+function blockem_display_item(App $a, array &$b = NULL) {
```

Comment:
Please use lowercase for `null`, `true` and `false`.

Reply:
Should I change all 20 `NULL` findings?

Reply:
Oh, except `gettext.php` (3rd-party lib).